app/src/main/res/layout/screen_patient_summary.xml

```diff
@@ -203,32 +203,32 @@
     android:paddingTop="@dimen/spacing_8"
     android:paddingBottom="@dimen/spacing_8">
 
-    <com.google.android.material.button.MaterialButton
-      android:id="@+id/doneButton"
-      style="?attr/materialButtonStyle"
+    <org.simple.clinic.widgets.ProgressMaterialButton
+      android:id="@+id/doctorButton"
       android:layout_width="0dp"
       android:layout_height="wrap_content"
       android:layout_marginStart="@dimen/spacing_8"
       android:layout_marginEnd="@dimen/spacing_8"
       android:layout_weight="1"
-      android:text="@string/patientsummary_finish_saving_patient"
+      android:text="@string/patientsummary_contact_doctor"
+      android:textAllCaps="true"
```

Comment:
I think I missed this during initial review, we don't have to use `textAllCaps` attr. Using the default material button style will automatically do this for us `style="?attr/materialButtonStyle" `